core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller.rb

```diff
@@ -116,6 +135,18 @@ def find_or_create_compose_file
     end
   end
 
+  def create_temporary_compose_file
```

Comment:
Let's move this into a service:

```
UffizziCore::ComposeFileService.create_temporary_compose(
  resource_project,
  current_user,
  compose_file_params,
  dependencies_params[:dependencies],
)
```

Reply:
resolved in this MR https://github.com/UffizziCloud/uffizzi_app/pull/124